prime-router/src/main/kotlin/Submission.kt

```diff
@@ -86,11 +196,18 @@ class DetailedSubmissionHistory(
      */
     val errorCount = logs.count { it.type == ActionLogLevel.error }
 
+    /**
+     * Destinations that actually had/will have data sent to.
+     */
+    val realDestinations: List<Destination> get() {
```

Comment:
See my comment above about maybe using an init { } instead of all these getters.

Reply:
I think leaving realDestinations as a getter makes sense, since it's just a single filter